server/protests.js

```diff
@@ -0,0 +1,84 @@
+const db = require('APP/db')
```

Comment:
I think this needs to be require('../db'); Same for server/users.js. The `APP/` path has been removed I think in the previous PR.